project/api/views.py

```diff
@@ -93,3 +94,48 @@ def create_request():
     except exc.IntegrityError as e:
         db.session.rollback()
         return jsonify(error_response), 400
+
+
+@request_blueprint.route("/edit_request/<requestid>", methods=["PUT"])
+def edit_request(requestid):
+    try:
+        put_data = request.get_json()
```

Comment:
Acho conveniente tirar essa parte do try except, pois podemos tratar esse erro de forma mais específica assim como acontece no POST https://github.com/Lend-it/Request/blob/devel/project/api/views.py#L65